### PR TITLE
Add artillery testing, trim old files

### PR DESCRIPTION
**Pull request by pafrias**
Preparing for full CRUD testing through artillery.

### CODE REVIEW

File: app/spec/nodebenchmark.js [hunk 1]
```diff
@@ -1,13 +1,22 @@
...
 const axios = require('axios');
 
-const randomNumber = n => Math.ceil(Math.random() * n);
+const randomFloat = n => Math.random() * n;
+
+const powerCurve = (max) => {
+  const range = Math.log(max);
+  const power = randomFloat(range);
+  return max - Math.floor(Math.exp(power));
+};
```
Comment (sharmoo) [inline app/spec/nodebenchmark.js:11]: good way to test the tail end of your data.

Comment (pafrias) [reply app/spec/nodebenchmark.js:11]: Yeah, I still have some versions of this floating in my code. I intend for them all be replaced by importing the helper function below.

File: app/spec/dbbenchmark.js [hunk 2]
```diff
@@ -34,19 +34,21 @@ async function readMongo(n) {
...
     const results = await readMongo(filesPerSec);
     const time = (Date.now() - start) / 1000;
     console.log(`${filesPerSec} reads in ${time} seconds : ${results} errors`);
+    total += time;
     if (time > 1) {
-      filesPerSec -= 10;
+      filesPerSec -= 5;
     } else if (time < 1) {
-      filesPerSec += 10;
+      filesPerSec += 5;
```
Comment (sharmoo) [inline app/spec/dbbenchmark.js:51]: what is with the ramping up and down of filesPerSec?

File: app/spec/nodebenchmark.js [hunk 3]
```diff
@@ -20,25 +29,33 @@ async function readComments(n) {
...
   while (testing) {
     tests += 1;
     const start = Date.now();
     const results = await readComments(filesPerSec);
     const time = (Date.now() - start) / 1000;
+    totalTime += time;
+    totalErrors += results;
+    totalReads += filesPerSec - results;
     console.log(`${filesPerSec} reads in ${time} seconds : ${results} errors`);
     if (time > 1) {
```
Comment (sharmoo) [inline app/spec/nodebenchmark.js:47]: wouldn't console.log every time slow things down?

File: app/spec/dbbenchmark.js (hunk 2 above)
Comment (pafrias) [reply app/spec/dbbenchmark.js:51]: Ah, this is a bad variable name. This should maybe be called readsPerSec, would do you think?

Comment (sharmoo) [reply app/spec/dbbenchmark.js:51]: ok. so you are trying to simulate the load rising and falling?

**Review by sharmoo — APPROVED** [review]

File: app/spec/nodebenchmark.js (hunk 3 above)
Comment (pafrias) [reply app/spec/nodebenchmark.js:47]: Only very slightly, it only console logs each test, not each read.

**Pull request merged**